client/src/components/screen/Friend.tsx

```diff
@@ -66,6 +92,10 @@ export default function Screen(): ReactElement {
     });
   };
 
+  const friends = data.friends.edges?.filter(
+    (x): x is NonNullable<typeof x> => x !== null,
+  ) || [];
```

Comment:
Could you kindly explain more about this code? Why do we need filter here?

Reply:
@hyochan The type of `data.friends.edges` is `(UserEdge | null)[] | null`. I filtered it so that `friends` is `UserEdge[]` type.